package.json

```diff
@@ -1,83 +1,84 @@
-    "version": "1.0.0",
-    "description": "Shuffle Andela members to perform various events.",
-    "author": "AndelaOSP",
-    "scripts": {
-        "prestart": "babel-node tools/startMessage.js",
-        "start": "npm-run-all --parallel test:watch open:src lint:watch",
-        "open:src": "babel-node tools/srcServer.js",
-        "lint": "node_modules/.bin/esw webpack.config.* src tools",
-        "lint:watch": "npm run lint -- --watch",
-        "test": "mocha --reporter progress tools/testSetup.js \"src/**/*.test.js\"",
-        "test:watch": "npm run test -- --watch"
-    },
-    "license": "MIT",
-    "dependencies": {
-        "babel-cli": "6.8.0",
-        "babel-core": "6.8.0",
-        "babel-loader": "6.2.4",
-        "babel-plugin-react-display-name": "2.0.0",
-        "babel-polyfill": "6.8.0",
-        "babel-preset-es2015": "6.6.0",
-        "babel-preset-react": "6.5.0",
-        "babel-preset-react-hmre": "1.1.1",
-        "babel-register": "6.8.0",
-        "bootstrap": "3.3.6",
-        "bulma": "^0.5.1",
-        "cheerio": "0.22.0",
-        "colors": "1.1.2",
-        "compression": "1.6.1",
-        "cross-env": "1.0.7",
-        "css-loader": "^0.23.1",
-        "dotenv": "^4.0.0",
-        "enzyme": "2.2.0",
-        "es6-promise": "^4.1.1",
-        "eslint": "2.9.0",
-        "eslint-plugin-import": "1.6.1",
-        "eslint-plugin-react": "5.0.1",
-        "eslint-watch": "2.1.11",
-        "eventsource-polyfill": "0.9.6",
-        "expect": "1.19.0",
-        "express": "4.13.4",
-        "extract-text-webpack-plugin": "1.0.1",
-        "file-loader": "^0.8.5",
-        "image-webpack-loader": "^3.3.1",
-        "isomorphic-fetch": "^2.2.1",
-        "jsdom": "8.5.0",
-        "jquery": "2.2.3",
-        "material-ui": "^0.18.7",
-        "mocha": "2.4.5",
-        "nock": "8.0.0",
-        "node-sass": "^4.5.3",
-        "node-uuid": "^1.4.8",
-        "npm-run-all": "1.8.0",
-        "open": "0.0.5",
-        
-        "react": "^15.6.1",
-        "react-addons-test-utils": "15.0.2",
-        "react-dom": "^15.6.1",
-        "react-redux": "4.4.5",
-        "react-router": "2.4.0",
-        "react-router-redux": "4.0.4",
-        "react-tap-event-plugin": "^2.0.1",
-        "redux": "3.5.2",
-        "redux-api-middleware": "^1.0.3",
-        "redux-immutable-state-invariant": "1.2.3",
-        "redux-mock-store": "1.0.2",
-        "redux-thunk": "2.0.1",
-        "rimraf": "2.5.2",
-        "sass-loader": "^6.0.6",
-        "style-loader": "^0.13.1",
-        "toastr": "2.1.2",
-        "url-loader": "0.5.7",
-        "webpack": "1.13.0",
-        "webpack-dev-middleware": "1.6.1",
-        "webpack-hot-middleware": "2.10.0"
-    },
-    "devDependencies": {},
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/coryhouse/pluralsight-redux-starter"
-    }
+  "name": "shuffle-box-react",
+  "version": "1.0.0",
+  "description": "Shuffle Andela members to perform various events.",
```

Comment:
Please use the description in the repo desc. "Shufflebox is an application that provides random selections and groupings for social events at Andela Kenya"